pkg/cluster/k8sres_test.go

```diff
@@ -2979,6 +2978,98 @@ func TestGenerateResourceRequirements(t *testing.T) {
 				ResourceLimits:   acidv1.ResourceDescription{CPU: "1", Memory: "2Gi"},
 			},
 		},
+		{
```

Comment:
Can we amend the other test to test for no huge pages set? Given its K8s 1.28 feature lets make sure its not set if not enabled.

Reply:
I've added a test to cover this